Britannia/localization/english/event_localization/BTWK_events_l_english.yml

```diff
@@ -1,25 +1,50 @@
-  BTWK_frontier_event.0201.a_brythonic:0 ""  # Unsure why Britons would rebuild the wall. Don't have idea for this
-  BTWK_frontier_event.0201.a_imperial:0 "We shall rebuild it to its former glory!"
-  BTWK_frontier_event.0201.a:0 "We will rebuild Hadrian's Wall."
-  
-  BTWK_frontier_event.0201.b_brythonic:0 "We cannot allow our people to divided any longer"
-  BTWK_frontier_event.0201.b_imperial:0 "Destroying the wall may placate the Northern Britons."  # Unsure why imperials would destroy the wall, other than to improve relations with local celts.
-  BTWK_frontier_event.0201.b:0 "We shall destroy the wall for materials."
-  BTWK_frontier_event.0201.c:0 "I will decide at another time."
-  BTWK_frontier_event.0201.c_tt:0 "#I The decision may be taken again at a later date. #!"
\ No newline at end of file
+  BTWK_frontier_event.0201:0 "The End of Hadrian's Wall"  # ROOT of decision trigger
+  BTWK_frontier_event.0201_desc:0 "The Rome of old has been gone from Britannia for many years. Behind it left great feats of engineering that have been left to crumble and decay, much like its successor rump states on the Isle. One of such constructions was #italic Hadrian's Wall#!. Constructed by Emporer Hadrian, it stood definitively as the northern-most border of Rome, designed to keep control of the Caledonians. Without Rome, it served, and still serves, no purpose. There was no border to defend - the land split between warring factions. \n\nAs such, denizens of the region around the wall had begun to repurpose the old stone to improve the infrastructure of their settlements, given it was so easily accessible. We will follow in their footsteps, and use its stone to prompt the development of the surrounding lands."
+  BTWK_frontier_event.0201.a:0 "Out with the old..."
+  BTWK_frontier_event.0201.a_tt:0 "All instances of the [special_buildings|E] #italic Hadrian's Wall#! or #italic Hadrian's Wall - Restored#! will be ruined."
+
+  BTWK_frontier_event.0202:0 "The End of Hadrian's Wall"  # Every other player
+  BTWK_frontier_event.0202_desc_header:0 "[GetScope(destroyed_hadrians_wall_char).Char.GetName] has ordered the demolition of Hadrian's Wall"
+  BTWK_frontier_event.0202_desc_header_vassal:0 "Our liege, [GetScope(destroyed_hadrians_wall_char).Char.GetName], has ordered the demolition of Hadrian's Wall."
+  BTWK_frontier_event.0202_desc_main:0 "\n\n [GetScope(destroyed_hadrians_wall_char).Char.GetHerHis] hopes that now the wall no longer serves any meaningful purpose, its stone could be repurposed for the developing settlements throughout [GetScope(destroyed_hadrians_wall_char).Char.GetHerHis] realm."
+  BTWK_frontier_event.0202_desc_imperial_like_destroyer:0 "[GetScope(destroyed_hadrians_wall_char).Char.GetFirstName] has done this despite [GetScope(destroyed_hadrians_wall_char).Char.GetHerHis] his heritage, from which is intrinsically linked to the momument. For this act, many of his kinsmen have been severely angered."
```

Comment:
despite [GetScope(destroyed_hadrians_wall_char).Char.GetHerHis] his heritage, from which is intrinsically linked to the momument. -> despite its immense historical and cultural importance.
For this act, many of his kinsmen have been severely angered. -> Many are furious at what they see as a callous disregard for the island's Roman heritage.

Reply:
Many are furious at what they see as a callous disregard for the island's Roman heritage -> Many are furious at what they see as a callous disregard - by one of their own kin no less - for the island's Roman heritage.